01-ActivityModel/activity-model/config/lad_codes.yaml

```diff
@@ -0,0 +1,8 @@
+################################################################################
+# Insert in this file the local authority codes (lad_codes) of your interest.
+# The default is Haringey.
+#
+################################################################################
+
+lad_codes:
+  - E09000014
```

Comment:
As people do not know where to find the lad code, I think this can be improving by providing small script where user can write the name and then get the code to make this modification.